webclient/uzivatel/admin.py

```diff
@@ -464,6 +470,70 @@ def get_readonly_fields(self, request, obj=None):
                 return fields + ("is_superuser",)
         return fields
 
+    def render_change_form(self, request, context, **kwargs):
+        object_id = request.resolver_match.kwargs.get("object_id")
+        user_account_history, user_account_other_records = self.get_histore_replated_records(object_id)
+        context.update(
+            {
+                "show_delete_history_button": True,
+                "object_id": object_id,
+                "user_account_history_exists": user_account_history.exists(),
+                "user_account_other_records_exists": user_account_other_records.exists(),
+            }
+        )
+        return super().render_change_form(request, context, **kwargs)
+
+    def get_urls(self):
+        urls = super().get_urls()
+        custom_urls = [
+            path(
+                "<path:object_id>/delete-history-records",
+                self.admin_site.admin_view(self.delete_history_records),
+                name="delete_history_records",
+            ),
+        ]
+        return custom_urls + urls
+
+    def get_histore_replated_records(self, object_id):
+        history = Historie.objects.filter(uzivatel=object_id)
+        user_account_history = history.filter(
```

Comment:
Nebylo by vázat to na situaci, kdy uživatel je referován přes `historie.vazba`? Jde o to, že pokud to navážeme na konkrétní transakce, bude to náročné udržovat (když přibude evidovaná transakce, snadno to sem zapomeneme přidat...).

Reply:
Může to tak být.

Reply:
Opraveno